### PR TITLE
Data mount

### PR DESCRIPTION
**Pull request by digitronik**
extra data mount
It will help to mount testing pages in selenium server

### CODE REVIEW

File: miqsel/server.py [hunk 1]
```diff
@@ -64,18 +64,23 @@ def vnc(self):
...
-    def start(self):
+    def start(self, **kwargs):
         """Start selenium container"""
 
         img = self.cfg["container"]["image"]
         name = self.cfg["container"]["name"]
+        mount_dir = self.cfg["container"]["data_dir"]
+
+        if mount_dir != "None":
+            kwargs.update({"volumes": {mount_dir: {"bind": "/usr/tmp", "mode": "ro"}}})
```
Comment (sbulage) [inline miqsel/server.py]: Strongly Suggest use either `/tmp` or `/var/tmp`

File: miqsel/config.py [hunk 2]
```diff
@@ -41,5 +41,8 @@ def config():
     cfg["container"]["server_port"] = click.prompt(
         "Selenium server running on port?", default=cfg["container"]["server_port"]
     )
+    cfg["container"]["data_dir"] = click.prompt(
+        "Extra data mount directory to '/urs/tmp'", default=cfg["container"]["data_dir"]
+    )
```
Comment (ganeshhubale) [inline miqsel/config.py:46]: Typo:
```'/urs/tmp'``` should be ```'/usr/tmp'```

File: miqsel/server.py (hunk 1 above)
Comment (digitronik) [reply miqsel/server.py]: `/tmp` in heavy container used for creating temp files and .x11 stuff. so its not valid.
about `/var/tmp` we can use but I think its sharing same like `/urs/tmp` right?

**Comment by digitronik** [comment]
@ganeshhubale @sbulage  done

File: miqsel/server.py (hunk 1 above)
Comment (sbulage) [reply miqsel/server.py]: Yes. @digitronik you are right, so instead of `/usr/tmp` use directly `/var/tmp`.

**Review by sbulage — APPROVED** [review]

**Comment by sbulage** [comment]
LGTM :+1:

**Pull request merged**